gnucash/report/business-reports/customer-summary.scm

```diff
@@ -97,8 +97,10 @@
 
 ;; ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(define-macro (addto! alist element)
-  `(set! ,alist (cons ,element ,alist)))
+(define-syntax addto!
+  (syntax-rules ()
+    ((addto! alist element)
+     (set! alist (cons element alist)))))
```

Comment:
Considering you're altering the same macro in 8 files, perhaps it should live somewhere in a common file ?

Reply:
Hmm, this seems a fundamental type function. Perhaps in utilities.scm?

Reply:
I've just tried to insert into utilities.scm and I don't think it's easy - https://www.gnu.org/software/guile/manual/html_node/R6RS-Incompatibilities.html - I keep getting an error whichever way I try to export a macro from utilities.scm... best leave as it is